git-gud.1

```diff
@@ -0,0 +1,70 @@
+.TH GIT-GUD 1 "19 Feb 2020" "version 0.0"
+.SH NAME
+git-gud \- get good at git with git-gud
+.SH SYNOPSIS
+.B git gud
+[\-\-version] [\-\-help]  
+<command> [<args>]
+.SH DESCRIPTION
+.B Git Gud
+teaches you how to use Git with fun interactive tutorials.  
+Use the following command to create a Git repository and start the first level:
+.B git\ gud\ init
+.SH OPTIONS
+.TP
+\fBstatus\fP
+Displays current level.
+.TP
+\fBinstructions\fP
+Displays instructions for current level.
+.TP
+\fBreset\fP
+Resets the current level.
+.TP
+\fBtest\fP
+Tests if current level is complete.
+.TP
+\fBload\ next\fP
```

Comment:
Move this by the other load commands.
There are also two more commands to add: `load prev` and `load previous`, which both load the previous commands

Reply:
I just update this and pushed the changes.